fiberzap/zap.go

```diff
@@ -81,8 +82,25 @@ func New(config ...Config) fiber.Handler {
 			stop = time.Now()
 		}
 
+		// Check if the logger has the appropriate level
+		s := c.Response().StatusCode()
+		var ce *zapcore.CheckedEntry
+		switch {
+		case s >= 500:
+			ce = cfg.Logger.Check(cfg.Levels[0], cfg.Messages[0])
```

Comment:
what happens if i change the series followers of the registered log level e.g.
```go
Levels: []zapcore.Level{zapcore.InfoLevel, zapcore.WarnLevel, zapcore.ErrorLevel},
```
or i remove some
```go
Levels: []zapcore.Level{zapcore.ErrorLevel},
```


currently, the construct does not seem safe to me

- [ ] please also add some tests for it